CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.5.4] - 2024-11-28
```

Comment:
If we're going to bump the version number here, we should do it in the root plugin file `index.php` as well so the correct version displays in the admin.

It looks like we haven't done that for a while, as it's still showing v1.5.1.